mne/tests/test_epochs.py

```diff
@@ -76,6 +76,14 @@ def test_reject():
                   picks=picks, preload=False, reject='foo')
     assert_raises(ValueError, Epochs, raw, events, event_id, tmin, tmax,
                   picks=picks_meg, preload=False, reject=dict(eeg=1.))
+    with warnings.catch_warnings(record=True) as w:
+        Epochs(raw, events, event_id, tmin, tmax, picks=picks_meg,
+               preload=False, reject=dict(eeg=1.),
+               allow_missing_reject_keys=True)
```

Comment:
It's a property of the class so users can change it themselves before
calling the function of they want. The only reason to put it in the
constructor is because the reject properties can also be set during
construction.